doc/dev/custom_devices.rst

```diff
@@ -134,9 +134,13 @@ One can follow the ``catalyst/runtime/tests/third_party/CMakeLists.txt`` `as an
 Integration with Python devices
 ===============================
 
+There are two things that are needed in order to integrate with PennyLane devices:
+
+* Adding a ``get_c_interface`` method to your ``qml.QubitDevice`` class.
+* Adding a ``config`` class variable pointing to your configuration file.
```

Comment:
We need to describe a bit more about this `config` file further down in this guideline.

Reply:
At the moment, since we don't use all of the fields, it feels to me like the meaning of how these fields are used is undefined. For example, we say "full" operations, but we don't really check for them. The only information we actively use are the matrix gates and they are used as a way to specify that the gates should be decomposed / transformed into a `qml.QubitUnitary`. The name `matrix` does not really denote this information very well. I would rather leave it unspecified until there is an actual use for them. Perhaps we should at some point implement the new device API and we will see better how these will fit?

Reply:
I added some more documentation here https://github.com/PennyLaneAI/catalyst/pull/369/commits/1b9fba147d17e6e01308966d187892be9f1cd81f

Reply:
https://github.com/PennyLaneAI/catalyst/pull/369/commits/268e480adc2919ca6459840ce89730e41c151979 and https://github.com/PennyLaneAI/catalyst/pull/369/commits/f3f124d8722c56371e9073cd73be82b721260068